### DISCUSSION TITLE
Searchable Docs

### DISCUSSION THREAD

**Discussion by james0r** [alpinejs/alpine#1600] [3. Feature ideas]
As far as i can tell the docs aren't currently searchable. 

Something with live recommendations would be sweet. Something like how Bootstrap doc's search works [https://getbootstrap.com/docs/5.0/getting-started/introduction/](https://getbootstrap.com/docs/5.0/getting-started/introduction/)

**Comment by KevinBatdorf** [comment]
If you're a crazy person like me and checked the docs source code, you'll find he has a search box there but it's commented out. My guess is he just needs to finish setting it up

**Reply by james0r** [reply]
lol you out-crazied me. looking forward to it.

**Comment by jensolafkoch** [comment]
Definitely needed! I often give up in the docs itself and have to use a search engine ... Strange. :-)

**Comment by philwolstenholme** [comment]
https://docsearch.algolia.com seems pretty popular and is free for open-source projects. Tailwind uses it too :)
